lib/middlewares/auth.js

```diff
@@ -73,7 +72,7 @@ passport.use(
         return done('No existe el usuario', false);
       }
 
-      return done(null, usuarioBD);
+      return done(null, { ...usuarioBD.toJSON() });
```

Comment:
Je, están armando un objeto ({}) para luego desarmarlo (...), más vale pasarlo directamente:

```suggestion
      return done(null, usuarioBD.toJSON());
```